mlflow/system_metrics/metrics/network_monitor.py

```diff
@@ -7,8 +7,29 @@
 
 
 class NetworkMonitor(BaseMetricsMonitor):
+    def __init__(self):
+        super().__init__()
+        self._set_initial_metrics()
+
+    def _set_initial_metrics(self):
+        # Set initial network usage metrics. `psutil.net_io_counters()` counts the stats since the
+        # system boot, so to set network usage metrics as 0 when we start logging, we need to keep
+        # the initial network usage metrics.
+        network_usage = psutil.net_io_counters()
+        self._initial_receive_megabytes = network_usage.bytes_recv / 1e6
+        self._initial_transmit_megabytes = network_usage.bytes_sent / 1e6
```

Comment:
can we add in `__init__`?
```
self._initial_receive_megabytes  = 0
self._initial_transmit_megabytes = 0
```

Reply:
I see, `_set_initial_metrics` is called in `__init__`.

Reply:
yes!